analyzer/evmabi/README.md

```diff
@@ -1,13 +1,14 @@
 # EVM ABIs
+
 This directory contains a Go package with Ethereum-compatible smart contract
 application binary interfaces (ABIs) and the Solidity sources.
 
-These interfaces are stable, so we don't have to maintain/rebuild them much.
-We currently use the [Remix web IDE](https://remix.ethereum.org/) to compile
-the Solidity sources (e.g. `contracts/ERC20.sol`) into JSON files containing
-EVM bytecode and ABI records (e.g. `contracts/artifacts/ERC20.json`).
-This is not automated, so for other developers' convenience, check in the
-compiled JSON files.
+These interfaces are stable, so we don't have to maintain/rebuild them much. We
+currently use the [Remix web IDE](https://remix.ethereum.org/) to compile the
```

Comment:
Mostly out of curiosity: How did you reformat all the .md files? `prettier` for rewrapping and `markdownlint --fix` for the rest?

Reply:
That is correct + 4 manual fixes for:
2x MD041 (First line in a file should be a top-level heading)
2x MD013 (line length) I have wrapped some SQL statement with `<!-- markdownlint-disable MD013 --><!-- markdownlint-enable MD013 -->`